tools/py/neb_interpolate.py

```diff
@@ -305,6 +322,12 @@ def spline_resample(q, nbeads_old, nbeads_new, k=3):
                     exit(-1)
 
                 natoms = path[0].natoms
+                masses = path[0].m
+                # Check that the masses in all frames are the same
+                for atoms in path:
+                    if np.any(atoms.m != masses):
+                        print("Error: masses differ across XYZ frames")
```

Comment:
If a suggestion on what they should do can be added, it could help the user. Like, make sure...